packages/tests/src/teardown.ts

```diff
@@ -1,23 +1,46 @@
 import { LightNode } from "@waku/interfaces";
 import debug from "debug";
+import pRetry from "p-retry";
 
 import { NimGoNode } from "./index.js";
 
 const log = debug("waku:test");
 
-export function tearDownNodes(
+export async function tearDownNodes(
   nwakuNodes: NimGoNode[],
```

Comment:
nit: I see we do `tearDownNodes([nodeA], [nodeB]);`
type can be changed to `NimGoNode | NimGoNode[]` and condition inside made like here https://github.com/waku-org/js-waku/blob/b10c46b910511418a048d7092dfd8b500a71a931/packages/core/src/lib/filter/index.ts#L76

Reply:
Thanks, looks better with this change